src/ufoLib2/errors.py

```diff
@@ -1,5 +1,18 @@
 from __future__ import annotations
 
+from typing import Any, Optional
+
 
 class Error(Exception):
     """The base exception for ufoLib2."""
+
+
+class ExtrasNotInstalledError(Error):
+    """The extras required for this feature are not installed."""
+
+    def __init__(self, extras: str, *, chained: Optional[Exception] = None) -> None:
+        super().__init__(f"Extras not installed: {extras!r}")
+        self.chained = chained
```

Comment:
I wonder if assigning the exception to `self.__cause__ ` instead does the same? Maybe that would make the call def redundant.

Reply:
good point! I'll do that